src/test/java/com/netflix/simianarmy/basic/chaos/TestBasicChaosEmailNotifier.java

```diff
@@ -68,6 +68,41 @@ public void beforeMethod() {
     }
 
     @Test
+    public void testInvalidEmailAddresses() {
+        String[] invalidEmails = new String[] { "username",
+                                                "username@.com.my",
+                                                "username123@example.a",
+                                                "username123@.com",
+                                                "username123@.com.com",
+                                                "username()*@example.com",
+                                                "username@%*.com"};
+        basicChaosEmailNotifier = new BasicChaosEmailNotifier(new BasicConfiguration(properties), sesClient, null);
+
+        for (String emailAddress : invalidEmails) {
+            Assert.assertFalse(basicChaosEmailNotifier.isValidEmail(emailAddress));
+        }
+    }
+
+    @Test
+    public void testValidEmailAddresses() {
+        String[] validEmails = new String[] { "username-100@example.com",
+                                              "name.surname+ml-info@example.com",
```

Comment:
this is the case that this PR addresses and that fails if tested on master:
Gradle suite > Gradle test > com.netflix.simianarmy.basic.chaos.TestBasicChaosEmailNotifier.testValidEmailAddresses FAILED
    java.lang.AssertionError at TestBasicChaosEmailNotifier.java:101